src/lib/execution_plan.rs

```diff
@@ -323,13 +324,31 @@ fn is_workspace_flow(
 /// Creates an execution plan for the given step based on existing execution plan data
 fn create_for_step(
     config: &Config,
-    task: &str,
+    task: &TaskIdentifier,
     steps: &mut Vec<Step>,
     task_names: &mut HashSet<String>,
     root: bool,
     allow_private: bool,
 ) {
-    let task_config = get_normalized_task(config, task, true);
+    if let Some(path) = &task.path {
+        // this is refering to a task in another file
+        // so we create a proxy task to invoke it
+        let proxy_name = format!("{}_proxy", task.name);
+        let mut proxy_task = create_proxy_task(&proxy_name, true, false);
+
+        proxy_task.command = Some("cargo".to_string());
```

Comment:
why do we need command and args? those are set by the create_proxy_task already which also populates all the needed args, not just task name.